alerts-server/app/com/alexitc/coinalerts/parsers/FixedPriceAlertFilterParser.scala

```diff
@@ -58,16 +58,19 @@ class FixedPriceAlertFilterParser {
   }
 }
 
+object FixedPriceAlertFilterParser {
 
-case class Filter(key: String, value: String) {
-  override def toString: String = s"$key=$value"
-}
-object Filter {
-  def from(string: String): Option[Filter] = {
-    Option(string.split(":"))
+  case class Filter(key: String, value: String) {
```

Comment:
In a similar way to the `WhereCondition`, I'm not convinced that the `Filter` belongs to the `FixedPriceAlertFilterParser`.